tests/python/op/cudnn/test_cudnn_nn.py

```diff
@@ -267,7 +266,7 @@ def forward(self, m_x, m_m, m_v, m_w, m_b):
     check(m_b.grad, t_b.grad, rtol=rtol, atol=atol)
 
 
-@with_dialect(["cudnn", "tvm"])
+@with_dialect(["cudnn"])
```

Comment:
does it still work with tvm?

Reply:
I think so although I didn't verify. On the other hand, TVM dialect won't be tested in this case anyways.